chemprop/cli/convert.py

```diff
@@ -26,7 +26,7 @@ def add_args(cls, parser: ArgumentParser) -> ArgumentParser:
             "-o",
             "--output-path",
             type=Path,
-            help="The path to which the converted model will be saved. Defaults to '<current working directory>/<stem of input>_v2.ckpt'",
+            help="The path to which the converted model will be saved. Defaults to 'PATH/TO/V1_MODEL_v2.ckpt'",
```

Comment:
I think this may be a little misleading. I just tried this while reviewing the CLI tutorials. I ran `chemprop convert --input-path tests/data/example_model_v1_regression_mol.pt` from the top-level chemprop directory and expected the output to be in `tests/data/`, but it was actually in my current directory. I'm not sure why this is given that we use `args.input_path.stem` in the code, but we should consider updating this help string to clarify.

Reply:
`Path().stem` returns just the name of the file without parents or suffix. I was just incorrect when I made this change. Either way could be the default, but I'll leave it how it is and fix the help string.